net-im/psi/psi-9999.ebuild

```diff
@@ -47,6 +47,7 @@ DEPEND="
 	dev-qt/qtx11extras:5
 	dev-qt/qtxml:5
 	net-dns/libidn:0
+	net-libs/http-parser
```

Comment:
excellent! net-libs/http-parser has a subslot, can you check if psi links against versioned subslot? if yes, needs rebuilt on slot change (`:=` operator)

Reply:
I have to remember this rule, thanks :)
Yes Psi is buildable even with versions earlier than ones in portage.